interface.go

```diff
@@ -9,7 +9,7 @@ import (
 
 const (
 	// MaxRangeRequestSize defines the max amount of headers that can be handled/requested at once.
-	MaxRangeRequestSize uint64 = 512
+	MaxRangeRequestSize uint64 = 64
```

Comment:
Now I wonder how hard would it be to plumb into config. It should be possible to configure this per node or per network. E.g. in rollkit case 512 is better and they already have constant size headers.

Reply:
we could always turn it into a config param if it's really needed.